tests/build/multi-platform.go

```diff
@@ -411,8 +421,617 @@ var _ = framework.MultiPlatformBuildSuiteDescribe("Multi Platform Controller E2E
+						Expect(err).ShouldNot(HaveOccurred())
+
+						prTrStatus := &pipeline.PipelineRunTaskRunStatus{
+							PipelineTaskName: chr.PipelineTaskName,
+							Status:           &taskRun.Status,
+						}
+
+						if chr.PipelineTaskName == constants.BuildTaskRunName && prTrStatus.Status != nil && prTrStatus.Status.TaskSpec != nil && prTrStatus.Status.TaskSpec.Volumes != nil {
+							multiPlatformTaskName = chr.Name
+							for _, vol := range prTrStatus.Status.TaskSpec.Volumes {
+								if vol.Secret != nil && strings.HasPrefix(vol.Secret.SecretName, "multi-platform-ssh-") {
+									multiPlatformSecretName = vol.Secret.SecretName
+									return nil
+								}
+							}
+						}
+					}
+					return fmt.Errorf("couldn't find a matching step buildah-remote or ssh secret attached as a volume in the task %s in PipelineRun %s/%s", constants.BuildTaskRunName, testNamespace, pr.GetName())
+				}, timeout, interval).Should(Succeed(), "timed out when verifying the buildah-remote image reference in pipelinerun")
+			})
+
+			It("The multi platform secret is populated", func() {
+				Eventually(func() error {
+					_, err := f.AsKubeAdmin.CommonController.GetSecret(testNamespace, multiPlatformSecretName)
+					if err != nil {
+						return err
+					}
+					return nil
+				}, timeout, interval).Should(Succeed(), "timed out when verifying the secret is created")
+
+				// Get the instance id from the task so that we can check during cleanup
+				taskRun, err := f.AsKubeDeveloper.TektonController.GetTaskRun(multiPlatformTaskName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred())
+				instanceId = taskRun.Annotations["build.appstudio.redhat.com/cloud-instance-id"]
+				GinkgoWriter.Printf("INSTANCE ID: %s\n", instanceId)
+				Expect(instanceId).ShouldNot(BeEmpty())
+			})
+
+			It("that PipelineRun completes successfully", func() {
+				Expect(f.AsKubeAdmin.HasController.WaitForComponentPipelineToBeFinished(component, "", f.AsKubeAdmin.TektonController, &has.RetryOptions{Retries: 2, Always: true})).To(Succeed())
+				pr, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(componentName, applicationName, testNamespace, "")
+				Expect(err).ShouldNot(HaveOccurred())
+				//now delete it so it can't interfere with later test logic
+				Expect(f.AsKubeAdmin.TektonController.DeletePipelineRun(pr.Name, testNamespace)).Should(Succeed())
+			})
+
+			It("check cleanup happened successfully", func() {
+				Eventually(func() error {
+					instances, err := getIbmZDynamicInstances(dynamicInstanceTag)
+					if err != nil {
+						return err
+					}
+					if len(instances) != 0 {
+						return fmt.Errorf("instance is not cleaned up properly, current running instances: %v", instances)
+					}
+					return nil
+				}, timeout, interval).Should(Succeed(), "timed out when verifying that the remote host was cleaned up correctly")
+			})
+
+		})
+	})
+	Describe("ibmp dynamic allocation", Label("ibmp-dynamic"), func() {
```

Comment:
Nitpick: use full spell "Power PC" instead of "p" so that it reflects the meaning straightforward.

Reply:
added in the describe section